### PR TITLE
Add comment and likes controller and views

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Add controllers for comments and likes.
Milestone 2:  Add views for comments and likes.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,4 +8,27 @@ def show
...
+
+  def new
+    post = current_user.posts.new
+    render :new, locals: { post: post }
+  end
+
+  def create
+    post = current_user.posts.new(params.require(:post).permit(:title, :text))
+    @user = current_user
+    @posts = @user.posts
```
Comment (AshakaE) [inline app/controllers/posts_controller.rb]: - [ ] Please make use of strong params. see [reference](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      format.html do
+        redirect_to user_post_path(@post.author.id, @post.id) if @comment.save
+      end
+    end
+  end
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
+end
```
Comment (AshakaE) [inline app/controllers/comments_controller.rb:22]: Remember to keep params private for securitty reasons [reference](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)

- [ ] Kindly make comment_params private

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Hector096  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
Hi **@Hector096,** 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**